src/main/java/dan200/computercraft/shared/peripheral/speaker/SpeakerPeripheral.java

```diff
@@ -55,6 +55,11 @@ public BlockPos getPos()
         return m_speaker.getPos();
     }
 
+    public synchronized boolean madeSound()
+    {
+        return (m_clock - m_lastPlayTime <= 20) ;
```

Comment:
This doesn't need to be `synchronized` because we're exclusively reading from atomic fields. Additionally, I feel it'd be nicer to make that `madeSound(long ticks)` to get rid of that magical `20` constant.